examples/labelanchor-example.ts

```diff
@@ -101,6 +101,11 @@ class LabelAnchorRenderer extends Renderer {
         this._labelPass.uninitialize();
     }
 
+    protected onDiscarded(): void {
```

Comment:
same for labelelide-example

Reply:
Yeah, it may be executed conditionally. However, I'm still unsettled regarding all current initialization routines. My goal would be to move once-initializing code to the constructor or use the optimization of resource allocation more thorough.